nntrainer/qnn_context.h

```diff
@@ -0,0 +1,299 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 Debadri Samaddar <s.debadri@samsung.com>
+ *
+ * @file    cl_context.h
+ * @date    23 Feb 2024
+ * @see     https://github.com/nnstreamer/nntrainer
+ * @author  Debadri Samaddar <s.debadri@samsung.com>
+ * @bug     No known bugs except for NYI items
+ * @brief   This file contains app context related functions and classes that
+ * manages the global configuration of the current OpenCL environment. It also
+ * creates the OpenCL command queue and context.
+ */
+
+#ifndef __CL_CONTEXT_H__
+#define __CL_CONTEXT_H__
```

Comment:
It should be updated for `qnn_context.h`.
```suggestion
#ifndef __QNN_CONTEXT_H__
#define __QNN_CONTEXT_H__
```